plugin/fzf.vim

```diff
@@ -632,10 +632,21 @@ function! s:execute(dict, command, use_height, temps) abort
       return []
     endif
   elseif has('win32unix') && $TERM !=# 'cygwin'
-    let shellscript = s:fzf_tempname()
-    call writefile([command], shellscript)
-    let command = 'cmd.exe /C '.fzf#shellescape('set "TERM=" & start /WAIT sh -c '.shellscript)
-    let a:temps.shellscript = shellscript
+    if !exists('s:cygwin_317_or_above')
+      let uname_os = system('uname -o')
```

Comment:
I would use `uname -s` instead since `-o` is not always available (see https://stackoverflow.com/a/27776822/755334)